static/css/game-events.css

```diff
@@ -1,37 +1,69 @@
-.filters-container {
-    margin-bottom: 20px;
-    padding: 20px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    width: 20%;
+.form-group1 {
+    display: grid;
+    grid-template-columns: repeat(5 , 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 50px;
+    grid-row-gap: 0px;
 }
 
-.form-group {
-    margin-bottom: 10px;
-}
+.input-group1 { grid-area: 2 / 1 / 3 / 2; }
+.input-group2 { grid-area: 2 / 2 / 3 / 3; }
+.input-group3  { grid-area: 2 / 3 / 3 / 4; }
+.button1 { grid-area: 2 / 4 / 3 / 5; }
+.button2 { grid-area: 2 / 5 / 3 / 6; }
+.past-events-checkbox { grid-area: 3 / 1 / 3 / 2; }
+
 
-.input-group {
-    display: flex;
-    align-items: center;
-    position: absolute;
-    justify-content: center;
+.form-control {
     width: 40% !important;
 }
 
-.input-number {
-    width: 100px;
-    text-align: center;
+.button1, .button2, .button3 {
```

Comment:
I agree with Yftach. Maybe we should change the name to "Submit Button" or more relevant name.

Does button1 and button3 behave the same? If yes, maybe we should rename them to be the same name? And then if I create a button in the future, I'll use the name you give it here and then the css would be automaticly applied, without needing to touch anything.

Reply:
Fixed this